# Print numbers from 1 to 20 using while loop.

In [1]:
num = 2
while num <= 20:
    print(num)
    num += 2

2
4
6
8
10
12
14
16
18
20


# Print every 3rd number from 3 to 30.

In [2]:
i = 3
for i in range(3, 31, 3):
    print(i)

3
6
9
12
15
18
21
24
27
30


# Print the sum of even numbers from 2 to 20.

In [8]:
even_sum = 0
for i in range(2, 21, 2):
    even_sum += i
    print("sum of even_number : ", even_sum)

sum of even_number :  2
sum of even_number :  6
sum of even_number :  12
sum of even_number :  20
sum of even_number :  30
sum of even_number :  42
sum of even_number :  56
sum of even_number :  72
sum of even_number :  90
sum of even_number :  110


# Print the product of odd numbers from 1 to 9.

In [7]:
a = 1
for i in range(1, 10, 2):
    a *= i
    print("Product of odd number : ", a)

Product of odd number :  1
Product of odd number :  3
Product of odd number :  15
Product of odd number :  105
Product of odd number :  945


# Create a list of 5 names and print each name on a new line.

In [9]:
names = ["Ayesha", "Meerab", "Faiza", "Seerat", "Farwa"]
for name in names:
    print(name)

Ayesha
Meerab
Faiza
Seerat
Farwa


# Count how many letters 'a' are in the word "banana".

In [10]:
word = "banana"
count_a = word.count('a')
print("numbers of 'a's : ", count_a)

numbers of 'a's :  3


# Print only the negative numbers from a given list.

In [11]:
nums = [1, -2, 3, -4, 5, -6]
for n in nums:
 if n < 0:
    print(n)

-2
-4
-6


# Print only the numbers ending in 5 from 1 to 50.

In [12]:
for i in range(1, 51):
    if i % 10 == 5:
        print(i)

5
15
25
35
45


# Print the lenght of each word in list ["cat", "elephant", "bat"].

In [13]:
words = ["cat", "elephant", "bat"]
for word in words:
    print(len(word))

3
8
3


# Take 3 numbers as input from the user and print their average.

In [14]:
a = int(input("Enter first number:"))
b = int(input("Enter second number:"))
c = int(input("Enter third number:"))
average = (a + b + c) / 3
print("average:", average)

average: 25.666666666666668


# print the table of 5 in reverse (from 5x10 to 5x1).

In [15]:
for i in range(10, 0, -1):
    print(f"5 x {i} = {5 * i}")

5 x 10 = 50
5 x 9 = 45
5 x 8 = 40
5 x 7 = 35
5 x 6 = 30
5 x 5 = 25
5 x 4 = 20
5 x 3 = 15
5 x 2 = 10
5 x 1 = 5


# Use a for loop to print all characters in "Python Is Fun".

In [16]:
char = "Python is fun"
for char in "Python is fun":
    print(char)

P
y
t
h
o
n
 
i
s
 
f
u
n


# Create a list of numbers and print every second item.

In [17]:
numbers = [10, 20, 30, 40, 50, 60]
for i in range(1, len(numbers), 2):
    print(numbers[i])

20
40
60


# Take a word from the user and count how many consonants it has.

In [21]:
word = input("Enter a word:")
count = 0
for ch in word:
    if ch.lower() not in "a, e, i, o, u" and ch.isalpha():
        count += 1
        print("number of consonant:", count)

number of consonant: 1
number of consonant: 2
number of consonant: 3
number of consonant: 4
number of consonant: 5


# Print the cubes of numbers from 1 to 5.

In [22]:
for i in range(1, 6):
    print(f"{i} ^ 3 = ", i ** 3)

1 ^ 3 =  1
2 ^ 3 =  8
3 ^ 3 =  27
4 ^ 3 =  64
5 ^ 3 =  125


# Take a list of words and print those that start with 'a'.

In [23]:
word_list = ["apple", "banana","cherry", "ant", "avocado"]
for w in word_list:
    if w.startswith('a'):
        print(w)

apple
ant
avocado


# Use a loop to print numbers between 100 and 110 (excluding 105).

In [24]:
for i in range(100, 111):
    if i != 105:
        print(i)

100
101
102
103
104
106
107
108
109
110


# Print the first 5 positive multiples of 4.

In [25]:
for i in range(1, 6):
    print(4 * i)

4
8
12
16
20


# Print each word in the sentence "Coding is easy" on a new line.

In [26]:
sentence = "Coding is easy"
for word in sentence.split():
    print(word)

Coding
is
easy


# **Create a number list and print only even numbers using conditions in loop.**

In [27]:
numbers = [1, 2, 3, 6, 7, 8, 10]
for n in numbers:
    if n % 2 == 0:
        print(n)

2
6
8
10
